cmd/daemon/apis/v6.go

```diff
@@ -0,0 +1,43 @@
+package apis
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// Handle Flux events
+func HandleV6(config APIConfig) (err error) {
```

Comment:
Why is there a named returned value here? As I read it this can never fail so the `error` return value could be removed entirely?